scripts/build/build.mts

```diff
@@ -51,6 +52,12 @@ async function copyVenmic() {
 
 await Promise.all([
     copyVenmic(),
+    composeTrayIcons({
```

Comment:
as far as i understood, tray badge is not available on mac, right? if so, this copy should not happen on mac

Reply:
does building on mac imply targeting mac?

Reply:
usually - electron-builder will use the host OS

Reply:
yes! artifacts are built on their respective platform. cross-compiling is technically supported, but
- i don't think you can cross compile mac builds
- we don't cross-compile (and don't support it)